src/services/user-group/services/course-copy-service.js

```diff
@@ -70,7 +74,7 @@ class CourseCopyService {
 			homeworkModel.find({ courseId: sourceCourseId }).populate('lessonId'),
 			lessonsModel.find({ courseId: sourceCourseId }),
 		]).catch((err) => {
-			throw new GeneralError('Can not fetch data to copy this course.', err);
+			throw new BadRequest('Can not fetch data to copy this course.', err);
```

Comment:
why 400? If you assume the ids are invalid, at least change the error message as well to sound like a BadRequest...

Reply:
Ich prüfe hier nochmal. Der 2. Parameter ging hier verloren. Das kann an den alten error handling liegen, oder an dem GeneralError von feathers.

Reply:
Ich habe ein util eingefügt welches die error params als 2 Parameter für feathers aufbereitet. Das Problem ist das feathers keine realen errors als weitere inputs erlaubt. https://docs.feathersjs.com/api/errors.html#examples